### ISSUE TITLE
FR: pki management

### ISSUE THREAD

**Issue by castlemilk** [istio-ecosystem/istio-installer#44]
would it be possible to include the management of PKI material and its injection into ingress/egress gateways and potentially pluggable CA?

**Comment by costinm** [comment]
Yes, it would be a great feature to have, in particular for the MC case.

Contributions welcomed. 

Keep in mind the plan is to start adding some golang code - the scripts are mainly as examples and to help integrate the installer with CI/CD and other tools. We want various steps included in istioctl ( which can vendor any code we add in this repo or in operator repo ).
We can start with the CLI tool from istio/security - which can generate some certs. 

I'm also ok including openssl in the 'installer' image ( I am planning to have the installer and operator 
available as a docker image with any required tool - helm, kustomize, etc).

Re. ingress/egress - I believe we now support SDS, we should enable it by default and add tests for it.
